Qwen/Qwen2.5-VL-7B-Instruct/accuracy/model_card_tasks.yml

```diff
@@ -0,0 +1,31 @@
+# from https://huggingface.co/RedHatAI/Qwen2.5-7B-quantized.w4a16
+tasks:
+  - name: arc_challenge
+    metrics:
+      - name: acc_norm,none
+        value: 0.5939
```

Comment:
I think its 58.70

Reply:
same for the rest. I think the ones you included are for the original model, not the quantized.

Reply:
My bad, I just realized that the source of values is from the link but it is for the original model,.